### PR TITLE
1028, 1029, 1030 Accessibility corrections for Reports AB#46056, AB#46083, AB#45387

### PR DESCRIPTION
**Pull request by csmith-he**
[AB#46056](https://hedev.visualstudio.com/76e71f30-b1ad-438a-a096-89ac98712f4e/_workitems/edit/46056)  - issue 1029
[AB#46083](https://hedev.visualstudio.com/76e71f30-b1ad-438a-a096-89ac98712f4e/_workitems/edit/46083) - issue 1028
[AB#45387](https://hedev.visualstudio.com/76e71f30-b1ad-438a-a096-89ac98712f4e/_workitems/edit/45387) - issue 1030

- Keyboard accessibility added to Geometry Metatdata in Location scene
- Keyboard accessibility added to edit/delete buttons in reports
- Contrast errors fixed in all reports
- Contrast errors fixed in all details reports

### CODE REVIEW

**Comment by csmith-he** [comment]
@phudson-he - Again, could you quickly act as my Dev peer reviewer for this, please?

File: arches_her/templates/views/components/reports/bibliographic-source.htm [hunk 1]
```diff
@@ -78,10 +78,10 @@ <h2 class="aher-report-section-title">
                                                 <td data-bind="text: part"></td>
                                                 <td class="aher-table-control">
                                                     <div data-bind="if: $parent.cards.sourceNames">
-                                                        <a data-bind="click: function() {$parent.editTile(tileid, $parent.cards.sourceNames)}">
+                                                        <a tabindex="0" data-bind="click: function() {$parent.editTile(tileid, $parent.cards.sourceNames)}" onkeyup="if(event.which == 13 || event.keyCode == 13){ $(this).trigger('click'); }">
```
Comment (phudson-he) [inline arches_her/templates/views/components/reports/bibliographic-source.htm:81]: Need to add some href's

Comment (csmith-he) [reply arches_her/templates/views/components/reports/bibliographic-source.htm]: @phudson-he Requested changes made - I think I caught them all

**Review by phudson-he — APPROVED** [review]
All looks good

**Comment by csmith-he** [comment]
@Rehaan-Arif  - this is now ready for Release Candidate testing

**Review by Rehaan-Arif — APPROVED** [review]
Testing complete on these tickets

File: arches_her/templates/views/components/reports/scenes/resources.htm [hunk 2]
```diff
@@ -452,11 +452,12 @@ <h2 class="aher-report-section-title">
                             <tr>
                                 <td><a class="aher-table-link" data-bind="text: resource, attr: {href: resourceLink}"></a></td>
                                 <td class="aher-table-control">
+                                    <div data-bind="text: console.log($parent.cards.translation)"></div>
```
Comment (aj-he) [inline arches_her/templates/views/components/reports/scenes/resources.htm:455]: @csmith-he there should not be a console log in the template right?

**Review by aj-he — CHANGES_REQUESTED** [review]
Can you sort the console log issue in the template and then we're good to go.

File: arches_her/templates/views/components/reports/scenes/resources.htm (hunk 2 above)
Comment (csmith-he) [reply arches_her/templates/views/components/reports/scenes/resources.htm]: @aj-he - Correct.  I've removed it now

Comment (aj-he) [reply arches_her/templates/views/components/reports/scenes/resources.htm]: looks good now

**Pull request merged**